docs/conf.py

```diff
@@ -170,4 +172,18 @@
 ]
 
 
+# -- Options for Napoleon-------------------------------------------------
+napoleon_use_ivar = True
 
+# -- Options for Autodoc--------------------------------------------------
+def run_apidoc(_):
+    from sphinx.apidoc import main
```

Comment:
It would be nice to have documentation showing how/why this gets autodoc working  (e.g. why certain flags are chosen, etc.) in case things need to be changed in the future.